src/forms/TimeInput.jsx

```diff
@@ -159,6 +172,35 @@ class TimeInput extends React.Component {
 		return Math.max(Math.min(value, max), min);
 	}
 
+	/**
+	 * @description increases or decreases a value
+	 * @param {Object} target target from event object
+	 * @param {Boolean} isIncreasing if the value is increasing
+	 * @param {Boolean} bigStep if the number should increase or decrease in a higher interval
+	 */
+	updateValueByStep(target, isIncreasing, bigStep) {
+		const { min, max, value } = target;
+		const currentVal = parseInt(value, 10);
+		const step = bigStep ? 10 : 1;
+		const newValue = isIncreasing ? currentVal + step : currentVal - step;
+
+		if (newValue <= max && newValue >= min) {
+			return formatDigits(newValue);
+		}
+	}
+
+	onNumberKeyDown(e) {
+		const { name } = e.target;
+		if(e.keyCode == 38 || e.keyCode == 40) {
```

Comment:
always good to label your `keyCode`s with code comments or by using named constants